.github/workflows/tests.yml

```diff
@@ -75,31 +94,36 @@ jobs:
       - uses: nrwl/nx-set-shas@v4
       - name: Monorepo install
         uses: ./.github/actions/yarn-nm-install
-      - name: Monorepo build
-        uses: ./.github/actions/run-build
       - name: Run lint
-        run: yarn nx affected --target=lint --parallel --nx-ignore-cycles
+        run: yarn prettier:check
 
-  build:
-    name: 'build (node: ${{ matrix.node }})'
+  lint:
+    name: 'lint (node: ${{ matrix.node }})'
+    needs: [build] # build is needed so that imports can be verified
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node: [20, 22]
     steps:
       - uses: actions/checkout@v4
+        with:
```

Comment:
isn't this the default behavior ?

Reply:
Oops, no, but it should be removed anyway

Reply:
`0` meant we were fetching the entire git history, and we were using it in every job 😆

removing this everywhere could actually make a noticeable difference in setup speed, trying it out now